### ISSUE TITLE
Also probing other data types, such as byte, long, float etc.

### ISSUE THREAD

**Issue by avandalen** [avandalen/avdweb_scope#4]

**Comment by boseji** [comment]
Yes Multi Datatype would become a crucial feature for the long run. This can be achieved using a type switch. But the storage is a kind a difficult question since each data type supports a different space requirement.

**Comment by avandalen** [comment]
Yes storing multiple data types is important but makes it too complex for now. Let's do this if everything works fine. I have worked with the variant class before, maybe this is a handy solution for the circulairbuffer.
